testing/python_lib/test_failscale.py

```diff
@@ -1,6 +1,7 @@
 """Integration test base class for Forch"""
 import unittest
 import time
+import multiprocessing
```

Comment:
as per comment below, there should be no mention of "multiprocessing" in this file -- everything specific to that should be abstracted away somehow

Reply:
Changed